test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs

```diff
@@ -25,6 +25,9 @@
 using Xunit;
 using Xunit.Abstractions;
 
+
+/* Niescalona zmiana z projektu „OpenTelemetry.Instrumentation.W3cTraceContext.Tests(net6.0)”
```

Comment:
What's this?

Reply:
Oh, some dotnet format shenanigans, let me fix this

Reply:
Fixed, weird issue and sorry I didn't catch it (too many PRs to look at)